libctru/include/3ds/services/ac.h

```diff
@@ -90,3 +95,36 @@ Result ACU_GetLastErrorCode(u32* errorCode);
  * @param errorCode Pointer to output the error code to.
  */
 Result ACU_GetLastDetailErrorCode(u32* errorCode);
+
+/**
+ * @brief Prepares a buffer to hold the configuration data to start a connection.
+ * @param config Pointer to an acuConfig struct to contain the data.
+ */
+Result ACU_CreateDefaultConfig(acuConfig* config);
+
+/**
+ * @brief Sets something that makes the connection reliable.
+ * @param config Pointer to an acuConfig struct used with ACU_CreateDefaultConfig previously.
+ * @param area Always 2 ?
```

Comment:
"2?"

Reply:
is this a question or are you saying I should change the "Always 2" into "2?" ?

Reply:
latter